src/background/extension.ts

```diff
@@ -148,7 +149,7 @@ export class Extension {
             // Fix for Firefox Android
             return;
         }
-        chrome.commands.onCommand.addListener(async (command) => {
+        chrome.commands.onCommand.addListener(debounce(500, async (command) => {
```

Comment:
Is this addListener similar to the EventEmitter addListener? If so, should we move each event into it's own encapsulated function? It seems this is a catch-all function for any kind of event.

Reply:
> Is this addListener similar to the EventEmitter addListener?

EventEmitter is pure Node.js way of Events and doesn't correspond with this kind of events that are executed from internal code.



> If so, should we move each event into it's own encapsulated function? It seems this is a catch-all function for any kind of event.

I'm not sure why because if we will do this with moving to such encapsulated function more know as block-scoped functions we will our ability to call private functions `this.` whereby having the () => format won't do this.